### PR TITLE
[zh-cn] Update in /doc/setup/_index.md

### PR DESCRIPTION
**Pull request by yanrongshi**
update:  #34540

### CODE REVIEW

**Review by Sea-n — COMMENTED** [review]
Looks good!
/lgtm

File: content/zh-cn/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md [hunk 1]
```diff
@@ -47,23 +47,21 @@ the documentation for that version to see advice that is relevant for your clust
...
-基于 CRD 对象所创建的自定义资源可以是名字空间作用域的，也可以是集群作用域的，
-取决于 CRD 对象 `spec.scope` 字段的设置。
-
-与其它的内置对象一样，删除名字空间也将删除该名字空间中的所有自定义对象。
-CustomResourceDefinitions 本身是无名字空间的，可在所有名字空间中访问。
+当你创建新的 CustomResourceDefinition（CRD）时，Kubernetes API 服务器会为你所
+指定的每一个版本生成一个 RESTful 的 资源路径。CRD 可以是名字空间作用域的，也可以
+是集群作用域的，取决于 CRD 的 `scope` 字段设置。和其他现有的内置对象一样，删除
+一个名字空间时，该名字空间下的所有定制对象也会被删除。CustomResourceDefinition
+本身是不受名字空间限制的，对所有名字空间可用。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md:64]: The above text was just tuned by another PR. This one is change them back.

**Comment by tengqm** [comment]
/hold

@yanrongshi Please learn yourself some git command before committing your changes.

File: content/zh-cn/docs/setup/_index.md [hunk 2]
```diff
@@ -56,7 +56,23 @@ If you don't want to manage a Kubernetes cluster yourself, you could pick a mana
 There are also other standardized and custom solutions across a wide range of cloud and
 bare metal environments.
 -->
-可以[下载 Kubernetes](/releases/download/)，在本地机器、云或你自己的数据中心上部署 Kubernetes 集群。
+可以[下载 Kubernetes](/zh-cn/releases/download/)，在本地机器、云或你自己的数据中心上部署 Kubernetes 集群。
```
Comment (tengqm) [inline content/zh-cn/docs/setup/_index.md:59]:
```suggestion
你可以[下载 Kubernetes](/zh-cn/releases/download/)，在本地机器、云或你自己的数据中心上部署 Kubernetes 集群。
```

**Pull request closed without merging**